docs/quick-start/project-structure.md

```diff
@@ -0,0 +1,197 @@
+├── Fixtures
+├── Images
+├── Lib
+├── Navigation
+│   └── Styles
+├── Redux
+├── Sagas
+├── Services
+├── Themes
+└── Transforms
+```
+
+**Components**
+"Dumb" components are stored here. All data is passed into dumb component. These components are often used inside a "Container Component". Container components are described in more detail further on.
+
+**Components ── Styles**
+We separate component styles from component functionality. Use this folder to create and store style files that matches the naming of your components. For example, a component `AlertMessage.js` would have a matching styles file called `AlertMessageStyles.js`.
+
+**Config**
+All application specific configuration falls in this folder. For example, `DebugSettings.js` is used for development-wide globals and `ReactotronConfig.js` is used for Reactotron client settings.
+
+**Containers**
+A container is what they call a "Smart Component" in Redux. It is a component
+which knows about Redux. They are usually used as "Screens".
+
+Also located here are two special containers: `App.js` and `RootContainer.js`.
+
+`App.js` is first component loaded after `index.ios.js` or `index.android.js`. The purpose of this file is to setup Redux or any other non-visual "global" modules. Having Redux setup here helps with the hot-reloading process in React Native during development as it won't try to reload your sagas and reducers should your colors change (for example).
+
+`RootContainer.js` is the first visual component in the app. It is the ancestor of all other screens and components.
+
+You'll probably find you can get quite far in an Ignite app without even touching these two files. They, of course, belong to you, so when you're ready to add something non-visual, like Firebase, or something visual, like an overlay, you have places for these additions.
+
+**Containers ── Styles**
+This `/Containers/Styles` folder will house your container styles. Each container component will likely have a companion styles file, just like `/Components`.
+
+**Fixtures**
+TODO: Correct this, the description is outdated.
+
+All key API responses are housed here.
+
+These API responses can be used for several reasons.  _E.G._:
+* To bypass logins when building any screen of the application
+* To quickly test API parsing in unit tests
+* To separate Network from Data concerns while coding
+
+**Images**
+Static images used in your project are stored here.
+
+**Lib**
+At first glance, this could appear to be a "miscellaneous" folder, but we recommend that you treat this as proving ground for components that could be reusable outside your project.
+
+Maybe you're writing a set of utilities that you could use outside your project, but they're not quite ready or battle tested. This folder would be a great place to put them. They would ideally be pure functions and have no dependencies related to other things in your App folder.
+
+**Navigation**
+TODO: Update with information about new navigation.
```

Comment:
Right.  We're hoping `react-navigation` will be ready for prime time before Ignite launches.  It's not yet.  Everyone has their fingers crossed.